olumidae/WayFarer
# #166901949 Ensure users can sign up and their records are created in Postgres

**What does this PR do?**
Ensures user signup records go to the database

**Description of the task to be completed?**
Setting up user creation records into Postgres is functional
Hashing of user password before posting to the database

**How should this be manually tested?**
Clone the repo cd into it and run,
Using postman, test the POST /api/v1/auth/signup endpoint
Insert the required records
Check the users table in the Wayfarer database on Postgres

**Any background context you want to provide?**
Not applicable

**What are the relevant pivotal tracker stories?**
#166901949
